src/managed_data/api.py

```diff
@@ -298,12 +298,14 @@ def get_dataframe(
         data[column["id"]] = []
 
     for row in rows:
-        if "fields" not in row.keys():
-            continue
-
         data[row_id].append(row["hid"])
         data["Validation Status"].append(row["validationStatus"])
 
+        if "fields" not in row.keys():
+            for column in columns:
+                data[column["id"]].append(None)
+            continue
+
```

Comment:
if we have an empty row, we append None to all columns and hope for the best

Reply:
Why? Can a dataframe row not just be empty?

Reply:
no. we have to put something in it